git-hooks/post-merge

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env node
+/* eslint-disable no-console */
+
+const execa = require('execa');
+const chalk = require('chalk');
+const fs = require('fs');
+
+execa.shell('git diff-tree -r --name-only --no-commit-id HEAD@{1} HEAD')
+    .then(result => {
+        const changedFiles = result.stdout.split(',');
+
+        if (changedFiles.includes('.nvmrc')) {
+            const nodeVersion = fs.readFileSync('.nvmrc');
+
+            console.log(`${chalk.red(`This application now uses Node.js v${nodeVersion}. Switch to the latest version using \`nvm use\` or download from nodejs.org. Then run \`make reinstall\``)}`);
```

Comment:
maybe `nvm install` in this instance? if it's installed already it will just switch anyway

Reply:
just for sake of [nit-picking consistency](https://github.com/guardian/frontend/blob/master/tools/check-node-env.js#L48), maybe

> Frontend now requires Node v${nvmrcVersion}. 

? 

not a blocker though